nixos/modules/system/boot/loader/grub/grub.nix

```diff
@@ -327,6 +327,27 @@ in
         '';
       };
 
+      extraInstallCommands = mkOption {
+        default = "";
+        example = literalExample ''
+          # the example below generates detached signatures that GRUB can verify
+          # https://www.gnu.org/software/grub/manual/grub/grub.html#Using-digital-signatures
+          ''${pkgs.findutils}/bin/find /boot -not -path "/boot/efi/*" -type f -name '*.sig' -delete
+          mkdir gpgtmp
```

Comment:
This literal script might better use a temporary directory (maybe `mktemp -d`) so there is no risk of it leaving the directory if a power outage happens during its execution.

Reply:
I was wondering whether that'd break sandboxing in that case, but I assume it wouldn't since you proposed it, gonna commit in a moment.